pages/common/git-credential.md

```diff
@@ -0,0 +1,16 @@
+# git credential
+
+> Retrieve and store user credentials.
+> More information: <https://git-scm.com/docs/git-credential>.
+
+- Display credential information, retrieving the username/password from config files:
+
+`echo {{"url=http://example.com"}} | git credential fill`
+
+- Send credential information to configured credential helpers to store for later use:
```

Comment:
Perhaps this will read better?
```suggestion
- Send credential information to all configured credential helpers to store for later use:
```

....or would it change the meaning?

Reply:
No it wouldn't change the meaning.  They used the word "any" but I think "all" is better.